README.md

```diff
@@ -124,6 +124,21 @@ base_uri: "https://appcanary.com/api/v3"
 monitor_name: "my_monitor"
 ```
 
+### Heroku
+
+For Heroku rails deployments, everything should really "just work".
+
+1. Include the `appcanary` gem in your `Gemfile`.
+2. Ensure you have the `api_key` set to `ENV["APPCANARY_API_KEY"]`.
+3. Set the env var like this: `heroku config:set APPCANARY_API_KEY=xxxx -a yorapp`
+4. Once deployed, try `heroku run rake appcanary:check` to verify your dependencies.
+5. Optionally, set up a regular monitor update using the heroku scheduler:
+  1. `heroku addons:create scheduler:standard`
+  2. `heroku addons:open scheduler`
+  3. Add a job that executes `rake appcanary:update_monitor`
```

Comment:
When you type the command in 2, it opens a browser on the scheduler. It's pretty obvious how to add a job once you do that.

Screenshots and stuff?